### ISSUE TITLE
Add support for Android Auto

### ISSUE THREAD

**Issue by mjlabe** [dkanada/gelli#166]
A few changes are needed to the app's manifest so that Android Auto can discover and interact with the app's media browser service.

https://developer.android.com/training/cars/media/auto

**Comment by rightsaidfred99** [comment]
why not just use jellyfin android auto?

**Comment by ShiniGandhi** [comment]
> why not just use jellyfin android auto?

Because Jellyfin is missing some key features on Android Auto, like shuffling, viewing your favorites. As well as other features like caching, downloading music, etc.

And these things afaik aren't really high on their priority.

**Comment by dkanada** [comment]
Unfortunately, it's also not very high on my list since I don't own a device that supports Android Auto. I would be more inclined to add Chromecast support before I worked on Android Auto.

**Comment by rightsaidfred99** [comment]
Android Auto is pretty useless anyway.

I think an in car ui would be more useful.  Larger buttons, shuffle button etc...

**Comment by seniorm0ment** [comment]
I agree with @rightsaidfred99 and I have a suggestion open for that here #198
However I wouldn't be opposed to future compatibility with Android Auto (and Chromecast) if it can be done in a way to keep the app completely FLOSS (at least for the F-Droid release)

**Comment by spupuz** [comment]
> > why not just use jellyfin android auto?
> 
> Because Jellyfin is missing some key features on Android Auto, like shuffling, viewing your favorites. As well as other features like caching, downloading music, etc.
> 
> And these things afaik aren't really high on their priority.

i would also be pleased to have android auto support! :)

**Comment by MatthewMedic** [comment]
This is really simple to add, I have done it and it works fine on my car using a UConnect receiver (Jeep Gladiator)

Changes required are as follows: 

Bump minSdk in build.gradle to 23

Add these lines to the AndroidManifest.xml
`        <meta-data android:name="com.google.android.gms.car.application"
                android:resource="@xml/automotive_app_desc"/>
`

Add a file `automotive_app_desc.xml` to `app/src/main/res/xml` with the following contents
`<automotiveApp>
    <uses name="media"/>
</automotiveApp>
`

Note that to run on production head units the app must be distributed through the play store or shared through the internal track via the Google Developer Console, so you can't just build and deploy with these changes and expect it to work.

**Comment by spupuz** [comment]
Please add it as a feature would be awesome 👍

**Comment by woolmonkey** [comment]
> This is really simple to add, I have done it and it works fine on my car using a UConnect receiver (Jeep Gladiator)
> 
> Changes required are as follows:
> 
> Bump minSdk in build.gradle to 23
> 
> Add these lines to the AndroidManifest.xml ` <meta-data android:name="com.google.android.gms.car.application" android:resource="@xml/automotive_app_desc"/>`
> 
> Add a file `automotive_app_desc.xml` to `app/src/main/res/xml` with the following contents `<automotiveApp> <uses name="media"/> </automotiveApp> `
> 
> Note that to run on production head units the app must be distributed through the play store or shared through the internal track via the Google Developer Console, so you can't just build and deploy with these changes and expect it to work.
> 
> I'd make a PR but my git app is giving me a weird error when I try only for this repo and I don't have time to figure it out.

You can use it on auto if you enable unknown sources for android auto in AA developer mode.  I'll have to try this and see if it working.
